site/docs/pages/index.mdx

```diff
@@ -7,6 +7,30 @@ layout: landing
 showOutline: false
 ---
 
+<div id="blobs">
```

Comment:
BLOBS!!!!

great name!!!!

Also love how so clean is the code. great fucking job!!!